xir/parser.py

```diff
@@ -4,22 +4,21 @@
 from pathlib import Path
 
 import lark
+from lark import v_args
 
 from .decimal_complex import DecimalComplex
-from .program import Declaration, ObservableStmt, Program, Statement
+from .program import Declaration, ObservableStmt, Program, Statement, ObservableFactor
 from .utils import simplify_math
 
+ENUM_WIRES = "ENUM_WIRES"
```

Comment:
I don't think we really need this constant.
```suggestion
```

Reply:
changed the naming of this constant to be more clear on the purpose